speed-dreams/source-2.2.3/src/drivers/assistedhuman/Driver.cpp

```diff
@@ -24,7 +24,8 @@ Driver::Driver(int p_index, const char* p_name) : m_index(p_index), m_humanDrive
 void Driver::InitTrack(tTrack* p_track, void* p_carHandle, void** p_carParmHandle, tSituation* p_situation) {
     m_humanDriver.init_track(m_index, p_track, p_carHandle, p_carParmHandle, p_situation);
 
-    SMediator::GetInstance()->RaceStart(p_track, p_carHandle, p_carParmHandle, p_situation);
+    bool recordBB = SMediator::GetInstance()->GetPControlSettings().BBRecordSession;
+    SMediator::GetInstance()->RaceStart(p_track, p_carHandle, p_carParmHandle, p_situation, recordBB);
```

Comment:
why are you sending a value that can be read by the mediator. I would change the RaceStart function to just read the BBRecordSession instead of sending it as a parameter of a function